bundlewrap/items/pkg_dnf.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from pipes import quote
+
+from bundlewrap.exceptions import BundleError
+from bundlewrap.items import Item
+from bundlewrap.utils.text import mark_for_translation as _
+
+
+def pkg_install(node, pkgname):
+    return node.run("dnf -d0 -e0 -y install {}".format(quote(pkgname)))
+
+
+def pkg_installed(node, pkgname):
+    result = node.run(
+        "dnf -d0 -e0 list installed {}".format(quote(pkgname)),
+        may_fail=True,
+    )
+    if result.return_code != 0:
+        return False
+    else:
+        return True
+
+
+def pkg_remove(node, pkgname):
+    return node.run("dnf -d0 -e0 -y remove {}".format(quote(pkgname)))
+
+
+class DnfPkg(Item):
+    """
+    A package installed by dnf.
+    """
+    BLOCK_CONCURRENT = ["pkg_dnf"]
```

Comment:
Need to learn more about blocking. I'm worried that in a mixed mode (old and new bundles) this will break something?
But yes, it's oke to have both on one node, as it won't break anything, only adds an unnecessary check.